src/v/config/configuration.cc

```diff
@@ -254,12 +254,18 @@ configuration::configuration()
   , target_quota_byte_rate(
       *this,
       "target_quota_byte_rate",
-      "Target quota byte rate (bytes per second) - 2GB default",
+      "Target request size quota byte rate (bytes per second) - 2GB default",
       {.needs_restart = needs_restart::no,
        .example = "1073741824",
        .visibility = visibility::user},
       2_GiB,
       {.min = 1_MiB})
+  , target_fetch_quota_byte_rate(
+      *this,
+      "target_fetch_quota_byte_rate",
+      "Target fetch size quota byte rate (bytes per second) - disabled default",
+      {.needs_restart = needs_restart::no, .visibility = visibility::user},
+      std::nullopt)
```

Comment:
(discuss) I think there is some value in making the response TP rate limiting symmetric to request TP limiting. Request TP limit default is 2GB/s, and when it was introduced it was not 100% compatible to what had been before. However the 2GB/s is quite a huge limit and I don't think that many of our customers can ever face it. Please let me know if you think there are any pros for making it different in this case.

Reply:
I think that we can make this disabled by default to save previous behavior.
It is hard to understand how much customers really consume. Maybe somebody have multiple consumers with same client_id
If customer wants this limit, he can configure it